source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/integ.twoZonesNoProps.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ *  Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+/// !cdk-integ *
+import { App, Stack } from "@aws-cdk/core";
+import { LambdaToElasticSearchAndKibana } from "../lib";
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as defaults from '@aws-solutions-constructs/core';
+
+// Setup
+const app = new App();
+const stack = new Stack(app, defaults.generateIntegStackName(__filename), {});
+
+const lambdaProps: lambda.FunctionProps = {
+  code: lambda.Code.fromAsset(`${__dirname}/lambda`),
+  runtime: lambda.Runtime.NODEJS_12_X,
+  handler: 'index.handler',
+};
+
+new LambdaToElasticSearchAndKibana(stack, 'test-lambda-elasticsearch-kibana', {
+  lambdaFunctionProps: lambdaProps,
+  domainName: "twozonesandnoprops",
```

Comment:
You are right, I had assume that this is environment-agnostic because I didn't defined the account but it seems the deployment is linked to my account which makes it environment specified therefore deploying a VPC in 3 AZ by default. This does not apply to the unit tests because the fake accounts need to be defined in the stack instantiation. 

## Environment specified stacks: 
```
const stack = new cdk.Stack(undefined, undefined, {
    env: { account: "123456789012", region: 'us-east-1' },
  });
```

This deploys a VPC in 3 AZ 

## Environment agnostic stacks:
```const stack = new cdk.Stack();```

This deploys a VPC in 2 AZ